README.md

```diff
@@ -38,7 +38,6 @@ A maintained and comprehensive list of awesome Fuel resources!
 - [The Case for Modular Maxis](https://davidphelps.substack.com/p/the-case-for-modular-maxis)
 - [Understanding Celestia's Blood Brother "Fuel": What's the difference betweeten a modular execution layer and a L2?](https://medium.com/@Inw5sAJJIrks6IE/understanding-celestias-blood-brother-fuel-what-s-the-difference-between-the-modular-27cbb1e959ce)
 - [Ethereum Scalability: Why Everyone Should Be Thinking About Fuel](https://mirror.xyz/camiinthisthang.eth/WsXmoZkrkY-w4w05nzyFrfP4n4AWM4qhmsyawqKFNTM)
-- [Fuel. A modular blockchain that solves the trilemma](https://medium.com/@Inw5sAJJIrks6IE/understanding-celestias-blood-brother-fuel-what-s-the-difference-between-the-modular-27cbb1e959ce)
```

Comment:
@camiinthisthang I cannot find the article titled "Fuel. A modular blockchain that solves the trilemma" 🤔 - has it been taken down?

Reply:
@mohammadfawaz looks like the link was pointing to the wrong thing and that one was removed ... let me try to track down that post and I'll add it.